commit daaf429cfe75899504fa2d1fa16b5c2c4e29b70b (refs/changes/41/3541/1)
Author: Orlando Vazquez <ovazquez@gmail.com>
Date:   2018-03-05T14:03:30-08:00 (1 year, 7 months ago)
    
    TRITON-207 Disable sdc-docker copy out of stopped container test due to OS-6699

diff --git a/test/integration/cli-copy.test.js b/test/integration/cli-copy.test.js
index 4f8a2e0..de7f7ca 100644
--- a/test/integration/cli-copy.test.js
+++ b/test/integration/cli-copy.test.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright (c) 2017, Joyent, Inc.
+ * Copyright (c) 2018, Joyent, Inc.
  */
 
 /*
@@ -418,6 +418,16 @@ test('copy a file out of running container', function (tt) {
 
 
 test('copy a file out of stopped container', function (tt) {
+    var skip = true;
+
+    if (skip) {
+        tt.skip(
+            'skipping test (copy out of stopped container) '
+            + 'until OS-6699 fixed');
+        tt.end();
+        return;
+    }
+
     tt.plan(9);
 
     var fnbase = '/var/tmp';
